src/Symfony/Bridge/Doctrine/Tests/Validator/Constraints/UniqueEntityValidatorTest.php

```diff
@@ -104,7 +104,7 @@ protected function createEntityManagerMock($repositoryMock)
         ;
 
         $classMetadata = $this->createMock(
-            class_exists(ClassMetadataInfo::class) ? ClassMetadataInfo::class : ClassMetadata::class
+            class_exists(ClassMetadataInfo::class) ? ClassMetadataInfo::class : ClassMetadata::class,
```

Comment:
There is weird, as the signature of createMock is 

```
protected function createMock(string $originalClassName): MockObject
```

Reply:
the arguments list is multiline - `(` and `)` are on different lines, even if it is provided with single argument. Thus, we get a comma.

ref https://www.php-fig.org/per/coding-style/#26-trailing-commas

Reply:
Ok.. i'll get used to :)